Makefile.am

```diff
@@ -26,7 +26,6 @@ noinst_PROGRAMS    = # Empty
 ALL_TESTS          = # Empty
 
 include include.am
-include src/libmeasurement_kit/ext/include.am
```

Comment:
Now dependencies are explicitly committed, to simplify cross platform development. As such, they can be computed automatically and some other complexity is gone.